docs/docs/sourcing-from-databases.md

```diff
@@ -1,13 +1,13 @@
 ---
-title: Sourcing from Databases
+title: Abastecimiento de la base de datos
 ---
 
-The Gatsby ecosystem includes built-in connectors to several database systems. These include:
+El ecosistema de Gatsby incluye incorporados conectores a varios sistemas de bases de datos. Éstos incluyen:
```

Comment:
Looks good, but I'd change this:
From "`incorporados conectores`" to "`conectores incorporados`".